spec/classes/collectd_plugin_fscache_spec.rb

```diff
@@ -13,6 +13,7 @@
     let :facts do
       {
         osfamily: 'FreeBSD',
+        operatingsystem: 'FreeBSD',
```

Comment:
Unrelated change to fix intermittent test failures.

Reply:
could you migrate this to rspec puppet facts?

Reply:
Love to, but there don't appear to be any. https://github.com/camptocamp/facterdb/tree/master/facts/2.5